public/404.html

```diff
@@ -44,66 +72,61 @@ <h2>Page not found (404)</h2>
+                  </p>
+                </div>
+               </div>
             </div>
-            <div class="col-md-4">
-              <p class="small">
-                By making it easier to use energy usage data in the classroom, Energy Sparks supports schools in integrating energy issues into the curriculum.
-                The data, along with the school's record of energy-saving activities, helps schools to achieve Eco-Schools awards.
-              </p>
-              <p class="small">
-                For feedback and questions, please email <a href="mailto:hello@energysparks.uk">hello@energysparks.uk</a>. Or contact us on social media
-              </p>
-              <ul class="text-center list-inline">
-                <li class="list-inline-item">
-                  <a href="https://www.facebook.com/EnergySparksUK/" class="btn-social btn-outline"><i class="fa fa-fw fa-facebook"></i></a>
+            <div class="col-5">
+              <h6>More information</h6>
+              <div class="row">
+                <div class="col">
+                  <ul class="list-unstyled">
+                    <li><a href="/for-teachers">Energy Sparks for Teachers</a></li>
+                    <li><a href="/for-pupils">Energy Sparks For Pupils</a></li>
+                    <li><a href="/contact">Contact us</a></li>
+                    <li><a href="/enrol">Register your school</a></li>
+                  </ul>
+                </div>
+                <div class="col justify-center">
+                   <ul class="list-unstyled">
+                    <li><a href="/team">Team</a></li>
+                    <li><a href="/team#partners">Partners</a></li>
+                    <li><a href="/datasets">Open data</a></li>
+                    <li><a href="/privacy_policy">Privacy policy</a></li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+
+            <div class="col-3">
+              <h6>Follow Energy Sparks</h6>
+              <ul class="list-inline">
+                <li class="list-inline-item p-2">
+                  <a href="https://twitter.com/energy_sparks" class=""><i class="fab fa-twitter fa-2x"></i></a>
                 </li>
-                <li class="list-inline-item">
-                  <a href="https://twitter.com/energy_sparks" class="btn-social btn-outline"><i class="fa fa-fw fa-twitter"></i></a>
+                <li class="list-inline-item p-2">
+                  <a href="https://www.facebook.com/EnergySparksUK/" class=""><i class="fab fa-facebook-square fa-2x"></i></a>
                 </li>
-                <li class="list-inline-item">
-                  <a href="https://github.com/BathHacked/energy-sparks" class="btn-social btn-outline"><i class="fa fa-fw fa-github"></i></a>
+                <li class="list-inline-item p-2">
+                  <a href="https://github.com/BathHacked/energy-sparks" class=""><i class="fab fa-github fa-2x"></i></a>
```

Comment:
These icons don't seem to render on test.

Reply:
Possible the old 404 css file is still in the cache? Could that be causing problems?

Reply:
Or, I'm getting confused and it's actually my browser that's caching it because these assets don't go through cloudfront...

Reply:
Nah, they're missing for me at home too - the font files haven't loaded...